pymtl3/passes/sverilog/import_/ImportPass.py

```diff
@@ -567,7 +566,7 @@ def gen_signal_init_c( s, name, port ):
           ret[ idx ] = "  " + ret[ idx ]
 
     else:
-      ret.append('m->{name} = {deference}model->{name};'.format(**locals()))
+      ret.append(f'm->{name} = {deference}model->{name};')
```

Comment:
much cleaner!

Reply:
yeah most of these **locals are gone ... since they basically serve the same purpose as fstring